docs/partner_editable/data_format.adoc

```diff
@@ -179,16 +179,22 @@ The output returns a list of reading dates in which anomalies are marked on a da
 }
 ----
 
-==== Meter outage details
+==== Meter-outage details
+
+The endpoint returns a list of meters that delivered an error code instead of a valid reading. Data includes geographic information (latitude, longitude) which can be used to create a heat map of current meter outages, for example, to visualize an outage cluster.
+
+//TODO Shivansh, Is "heat map" correct above, or do we just mean "map"?
 
-Endpoint that returns a list of meter that delivered an error code instead of a valid reading. Data includes geograpihc information (latitude, longitude) which can be used to great a heat map of current meter outages to i.e. visualize outage cluster.
+startDateTime: Describes the start time of the time windows.
 
-startDateTime:Describes the start time of the time windows.
+//TODO Shivansh, should startDateTime, above, be start_date_time to match the table below?
 
-end_date_time: describes: Describes the end time of the time window.
+end_date_time: Describes the end time of the time window.
+
+//TODO Shivansh, Why are we  listing and describing these two items above when the table below does the same? I recommend deleting the above two lines and adding any relevant details from their descriptions to the table below.
```

Comment:
Sure will do. Do you want me to go ahead and make those changes or should i wait for your edits to avoid merge conflicts?

Reply:
Sascha has done this editing in a [commit](https://github.com/aws-quickstart/quickstart-aws-utility-meter-data-analytics-platform/commit/167776badaf447c38471e787a561aac52a340caa) after this.